src/test/Button.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import BackButton from '../components/BackButton';
+
+test('Clicking the BackButton should navigate to the home page', () => {
+  render(
+    <BrowserRouter>
+      <BackButton />
```

Comment:
- [ ]  Great job for adding some interactions between the components of your app(integration tests) :+1: As pointed out by the previous reviewer, I think you'll required to add more tests to the functionalities of your app. I think you'll required to also test interaction with the API. As you can see on the screenshot below. To avoid actual requests, you may need to mock the API and test it and also mock the connection to the Redux store.
- [ ] In order to make sure each unit of our application is working as intended, I think it's important to add unit tests for data that comes from the backend(API) and your components are displaying them.  As mentioned above, you'll need to pass some mock data(home_team_score, season, period, etc..) relevant to what you want to test and make sure the mock data renders successfully :+1: 
![Screenshot from 2023-10-20 18-35-39](https://github.com/enocol/Metrics-webapp/assets/63087491/efe4e95c-f74b-4463-982d-90bce38f1daf)

Reply:
There is a limite of how much a person can take. let student success decide. I have made changes to the test request. Project require if mobile only not desktop

Reply:
@FrederickMih I am not able to make any more changes to the project, so I have requested that student success look into this.

Reply:
> @FrederickMih There is a limit of how much a person can take. let student success decide. I have made changes to the test requested. Project require is mobile only not desktop.

Hi @enocol, I mentioned this as optional